mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MifosBaseActivity.java

```diff
@@ -122,9 +124,12 @@ public void hideProgress() {
 
     public void hideKeyboard(View view) {
         InputMethodManager inputManager = (InputMethodManager) this
-                 .getSystemService(Context.INPUT_METHOD_SERVICE);
-        inputManager.hideSoftInputFromWindow(getCurrentFocus().getWindowToken(), InputMethodManager
-                 .RESULT_UNCHANGED_SHOWN);
+                .getSystemService(Context.INPUT_METHOD_SERVICE);
+        View view1 = getCurrentFocus();
+        if (view1 != null) {
```

Comment:
The code can be made concise. Remove line 128 and use getCurrentFocus() directly in the if statement.